collections/concepts/applications-concept.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title:  Applications
+categories: concepts
+permalink: /concepts/applications/
+nav_order: 1
+has_children: true
+has_toc: false
+image:
+    path: https://cdn.basistheory.com/images/seo/guides-opengraph.png
+    width: 1200
+    height: 630
+---
+
+# What are Applications?
+
+Basis Theory’s Applications do not imply any specific implementation type. Applications could be a native app on a mobile device, a single-page application in a browser, or a regular web application that executes on a server or in a serverless environment.
+
+## How It Works
+
+Applications within Basis Theory have the following properties and characteristics:
+
+### Name
+
+The name of your Application gives you clarity on what context it is being used for, for example, “Acme Billing App” or “Acme Mobile App”. 
+
+### Application Types
+
+- Server-to-Server
+    - Used with a traditional web application that runs their application logic primarily on a server or in a serverless environment (e.g. NodeJs, .NET, Java, or Ruby). This application type has the most permissions available as the API Key will be private and secured within a server environment.
+- Elements
+    - Used with browser-based applications (e.g. Vanilla Javascript, React, Angular, etc) using the Basis Theory Elements to collect and secure data.
+- Client-side
+    - Used for Browser, Mobile, or Desktop applications running natively on a device (e.g. iOS, Android, Windows, etc) where the application is Collecting the data itself a protecting with Basis Theory via an API call. Applications created with this type have limited permissions to enable using the API Key in publicly available code.
+- Management
+    - Used with scripts or the Basis Theory Terraform (request access here) modules to configure your [Tenant’s](https://guides.basistheory.com/concepts/tenants/) settings (Applications, Reactors, etc) without logging into our UI Portal.
+
+### Permissions
+
+Each Application Type has different levels of permission available for the application. These permissions range from enabling the ability to Create Tokens, Use Tokens with our Proxy or Reactors, all the way to the ability to create new applications. You can find a detailed list of all of our permissions for all of our [Application types here.](https://docs.basistheory.com/api-reference/#permissions-permission-types)
+
+### API Key
+
+Your Application’s API Key is used to authenticate your systems to the Basis Theory platform. You will use this with our API via our `BT-API-KEY` header or into our Elements SDK’s to grant access to collect and secure data from the browser.  These API Keys follow a format similar to `key_4qUtg83907AnDem90aJSzcN`.

```

Comment:
```suggestion
Your Application’s API Key is used to authenticate your systems to the Basis Theory platform. The API Key can be used to make authenticated requests to our API directly via the `BT-API-KEY`, using Hosted Elements, or using the Basis Theory SDK. These API Keys follow a format similar to `key_4qUtg83907AnDem90aJSzcN`.

```

Trying to capture all the ways an API Key can be used to interact with BT. We could add links to our API Docs and Hosted Elements. Do we have somewhere we could link to cover all the SDKs we offer?